beast/physicsmodel/tests/test_stellar_grid_weights.py

```diff
@@ -1,15 +1,82 @@
 import numpy as np
 
-from beast.physicsmodel.grid_weights_stars import compute_distance_grid_weights
+from beast.physicsmodel.grid_weights_stars import *
+
+
+def test_bin_boundaries():
+    """
+    Test bin boundaries
+    """
+    bin_centers = np.array([1, 2, 5, 10, 50])
+    weights = compute_bin_boundaries(bin_centers)
+    expected_weights = [0.5, 1.5, 3.5, 7.5, 30.0, 70.0]
+    np.testing.assert_allclose(
+        weights, expected_weights, err_msg=("\nStellar bin boundaries error\n")
```

Comment:
Mainly curious.  Why the `\n` before and after the error message text?

Reply:
Just seemed easier to read, no other reason.

Reply:
I would remove it as I think this will make it harder to handle the error messages if someone wants to manage them differently than just outputting them to the screen.

Reply:
Yeah I understand. I'll make the change!